src/malloc_wrapper.cc

```diff
@@ -99,7 +99,7 @@ void *__wrap_malloc(size_t size)
```

Comment:
[llvmlibc-callee-namespace]: 'backtrace' must resolve to a function declared within the '__llvm_libc' namespace